packages/payment-detection/test/any/any-to-any-proxy-info-retriever.test.ts

```diff
@@ -2,22 +2,102 @@
 /* eslint-disable no-magic-numbers */
 import { PaymentTypes } from '@requestnetwork/types';
 import { CurrencyManager } from '@requestnetwork/currency';
-import AnyToErc20ProxyInfoRetriever from '../../src/any/any-to-erc20-proxy-info-retriever';
+import AnyToAnyProxyInfoRetriever from '../../src/any/any-to-any-proxy-info-retriever';
 import { ethers } from 'ethers';
 
 const erc20LocalhostContractAddress = '0x38cF23C52Bb4B13F051Aec09580a2dE845a7FA35';
 const conversionProxyContractAddress = '0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4';
-const erc20FeeProxyContractAddress = '0x75c35C980C0d37ef46DF04d31A140b65503c0eEd';
 const paymentReferenceMock = '01111111111111111111111111111111111111111111111111';
 const acceptedTokens = [erc20LocalhostContractAddress];
 const USDCurrency = CurrencyManager.getDefault().from('USD')!;
-
+const conversionAbi = [
```

Comment:
any reason not to use the artifacts?

Reply:
I agree, for future tests, it would be better to base this on artifacts